src/scss/2022/components/_circle.scss

```diff
@@ -0,0 +1,50 @@
+.cmp-circle {
+  position: fixed;
+  --distance: 2rem;
+  top: var(--distance);
+  left: var(--distance);
+  right: var(--distance);
+  bottom: var(--distance);
+  z-index: 100;
+  clip-path: circle();
+  
+  @supports (mix-blend-mode: hard-light) {
+    z-index: 1000;
+    pointer-events: none;
+    mix-blend-mode: hard-light;
+  }
+
+  &__hero-1 {
+    outline: 1px solid cyan;
```

Comment:
Is the outline a leftover development aid or should it be visible in the browser?

Reply:
That is exactly what that is. Thank you!